### PR TITLE
[DE-582] Add campaign auto save

### PR DESCRIPTION
**Pull request by Cromeror**

### CODE REVIEW

File: src/components/SingletonEditor.tsx [hunk 1]
```diff
@@ -41,6 +42,8 @@ export const SingletonDesignContext = createContext<ISingletonDesignContext>({
   editorState: { isLoaded: false, unlayer: undefined },
 });
 
+const AUTO_SAVE_INTERVAL = 6000;
```
Comment (andresmoschini) [inline src/components/SingletonEditor.tsx:45]: What do you think, at least for the beginning, to increase this value? What do you think about 30 segs?

File: src/components/SingletonEditor.tsx [hunk 2]
```diff
@@ -113,7 +121,7 @@ export const useSingletonEditor = (
           onLoadEventListener
         );
         editorState.unlayer.removeEventListener(
-          "design:loaded",
+          "design:updated",
```
Comment (andresmoschini) [inline src/components/SingletonEditor.tsx:124]: Was it a bug in the previous code?

Comment (Cromeror) [reply src/components/SingletonEditor.tsx:124]: Yes, it wasn't unmounting properly.

File: src/components/SingletonEditor.tsx (hunk 1 above)
Comment (Cromeror) [reply src/components/SingletonEditor.tsx:45]: I agree!!

**Pull request merged**